hack/pre-release.sh

```diff
@@ -148,6 +158,126 @@ update_OCP_version() {
   fi 
 }
 
+# This function handles updating the versions in GitHub
+# takes 3 arguments
+#
+# 1st arg: base branch to operate on 
+# 2nd arg: stream to update. Can be x, y or z to represent the x y or z stream.#
+# 3rd arg: current WMCO version
+github_update(){
+  if [ "$#" -ne 3 ]; then
+    echo incorrect parameter count for github_update $#
+    return 1
+  fi
+
+  base_branch=$1
+  stream=$2
+  wmco_version=$3
+
+  # check if the git branch is dirty, and exit if it is
+  check_git_branch
+
+  new_branch=$(create_git_branch_list $stream-stream-bump $base_branch)
+  switch_git_branch $base_branch $new_branch
+
+  # Locate and store the WMCO version from the makefile
+  # finds "WMCO_VERSION ?=" and selects text following it
+  update_WMCO_version $stream $wmco_version
+  update_OCP_version $stream $wmco_version
+
+  # check to see if the opeator-sdk version is correct
+  operator_sdk_current_version=$(operator-sdk version |  grep -oP 'operator-sdk version: "\K[^"]+' )
+  if [[ "$operator_sdk_current_version" != "$OPERATOR_SDK_VERSION" ]]; then
+    echo -e "${PINK}operator-sdk $OPERATOR_SDK_VERSION must be installed.${RESET}"
+    echo -e "${PINK}exiting.${RESET}"
+    git checkout $initial_branch
+    exit
+  fi
+
+  git submodule update --init --recursive
+  make bundle
+
+  commit_message="[$base_branch] Update version to $wmco_version
+
+  This commit was generated by hack/pre-release.sh"
+  generate_commit "$commit_message" build bundle Makefile config
+
+  # Display the newly created branches with push instructions
+  push_message=$(display_push_message "$new_branch")
+  echo
+  echo -e "${PINK}${push_message}${RESET}"
+  echo
+}
+
+# This function handles updating the versions in dist_git. Expects you to be connected to the VPN 
```

Comment:
This function just generates the patch. The user still needs to apply and push the commit.